server/tests/io/feed_parsers/belga_afp_newsml_1_2_test.py

```diff
@@ -41,6 +41,7 @@ def test_content(self):
             {'name': 'trials', 'qcode': '02008000', 'scheme': 'iptc_subject_codes'},
             {'name': 'AMN-TFG-1=AMW', 'qcode': 'AMN-TFG-1=AMW', 'scheme': 'of_interest_to'},
             {'name': 'EUA-TFG-1=EUA', 'qcode': 'EUA-TFG-1=EUA', 'scheme': 'of_interest_to'},
+            {'name': 'EUA-TFG-1=EUA', 'qcode': 'EUA-TFG-1=EUA', 'scheme': 'of_interest_to'},
```

Comment:
I think it's a duplication of a dict above. Why do we need it?

Reply:
because we delete code to remove duplicated subject items, fixed in https://github.com/superdesk/superdesk-belga/pull/124/commits/9f9405f10a2c5438ad75e470c4999996f3dea071